### PR TITLE
set primitive owner

### PR DESCRIPTION
**Pull request by Fredaly**
Change primitive component owner without changing attachment tree

### CODE REVIEW

File: Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp [hunk 1]
```diff
@@ -146,3 +147,10 @@ void UCoreExtHelperBlueprintLibrary::SetEditorOnly( AActor * actor, bool editor_
 {
     actor->bIsEditorOnlyActor = editor_only;
 }
+void UCoreExtHelperBlueprintLibrary::SetPrimitiveComponentOwner( UPrimitiveComponent * component, AActor * actor )
+{
+    if ( component != nullptr || actor != nullptr )
```
Comment (TheEmidee) [inline Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp]: &&, not ||

File: Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp [hunk 2]
```diff
@@ -146,3 +147,10 @@ void UCoreExtHelperBlueprintLibrary::SetEditorOnly( AActor * actor, bool editor_
 {
     actor->bIsEditorOnlyActor = editor_only;
 }
+void UCoreExtHelperBlueprintLibrary::SetPrimitiveComponentOwner( UPrimitiveComponent * component, AActor * actor )
```
Comment (TheEmidee) [inline Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp]: Can this be done on UActorComponent ?

File: Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp (hunk 1 above)
Comment (Fredaly) [reply Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp]: done

File: Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp (hunk 2 above)
Comment (Fredaly) [reply Source/CoreExtensions/Private/BlueprintLibraries/CoreExtHelperBlueprintLibrary.cpp]: yes, done

**Review by ThibNach — APPROVED** [review]

**Review by Frickerson — APPROVED** [review]

**Pull request merged**